srcs/mainConfigFile.cpp

```diff
@@ -6,68 +6,92 @@ int main(int ac, char **av, char **env) {
         std::cout << std::boolalpha;
         std::cout << "Configuration file name: " << av[1] << std::endl;
         std::cout << "-----------------------------------------------------------------------------------" << std::endl;
-        data::ConfigFile configFile(av[1]);
-        data::Server serverData = configFile.getServerData();
+        data::ConfigFile configFileData(av[1]);
+        std::map<data::Server*, std::vector<data::Location> > const & serverDataMap = configFileData.getServerDataMap();
         std::cout << "-----------------------------------------------------------------------------------" << std::endl;
 
-//        std::cout << CYN << "Has server block: [" << configFile.getServerData().hasServerBlock() << "]" << BACK << std::endl << std::endl;
-        std::cout << std::endl << "Starting server block" << std::endl;
+        if (not serverDataMap.empty()) {
+            std::map<data::Server*, std::vector<data::Location> >::const_iterator it_server;
+            for (it_server = serverDataMap.begin(); it_server != serverDataMap.end(); it_server++) {//todo check cbegin
```

Comment:
how to access the key of the map (with the first method), which will give us the server block(s).